2203/2/versus.rb

```diff
@@ -0,0 +1,22 @@
+require 'optparse'
+require 'terminal-table'
+require './russian_declensions.rb'
+require './bad_words_counter.rb'
+
+OptionParser.new do |options|
+  options.on('--top-bad-words=') do |number|
+    table = Terminal::Table.new do |row|
+      BadWordsCounter.sorting_bad_words[0...number.to_i].to_h.each do |name, bad_words_count|
```

Comment:
you are still calling TopWordsStatistic for each rapper twice :)

Reply:
@anatoliliotych fixed